tests/hpc/rasdaemon.pm

```diff
@@ -60,6 +60,9 @@ sub run {
     my $rt = zypper_call('in rasdaemon');
     test_case('Installation', 'rasdaemon', $rt);
 
+    # Skip functional tests on ppc64le and Tumbleweed
```

Comment:
please add 'why' we skip as otherwise this comment  isn't that useful?

Reply:
better?